test/parallel/test-debugger-util-regression.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+const path = require('path');
+const spawn = require('child_process').spawn;
+const assert = require('assert');
+
+const common = require('../common');
+
+const fixture = path.join(
+  common.fixturesDir,
+  'debugger-util-regression-fixture.js'
+);
+
+const args = [
+  'debug',
+  fixture
+];
+
+const proc = spawn(process.execPath, args, { stdio: 'pipe' });
+
+function cleanup() {
+  process.exitCode = 2;
+  proc.kill();
+}
+
+const timer = setTimeout(cleanup, common.platformTimeout(2000));
+
+let stdout = '';
+let stderr = '';
+
+let nextCount = 0;
+
+proc.stdout.on('data', (data) => {
+  stdout += data;
```

Comment:
If you're going to use string concatenation, call `proc.stdout.setEncoding('utf8')` first.  Ditto for stderr.